src/TweetProcessor.js

```diff
@@ -257,23 +293,24 @@ class TweetProcessor {
       promise = promise.then(() => { return this._twitter.profile.update(tweet.user) });
     }
 
-    promise.then( () => {
+    return promise.then( () => {
       if(typeof rooms == "string") {
         rooms = [rooms];
       }
       rooms.forEach((roomid) => {
         this._storage.room_has_tweet(roomid, tweet.id_str).then(
           (room_has_tweet) => {
             if (!room_has_tweet) {
+              const realUserId = '_twitter_'+tweet.user.id_str;
```

Comment:
Actually, this is me being lazy and awful because I prefer my operators spaced out. I think a rule is needed.